tests/pytests/test_dialect.py

```diff
@@ -4,6 +4,8 @@
 from includes import *
 import numpy as np
 
+MAX_DIALECT = 3
```

Comment:
Why not extract it from the server with some debug command so we will never have to update it again?

Reply:
DEBUG is not available on cluster.
Updating this once we introduce a new dialect is not ideal but acceptable, no?

Reply:
You have `_FT.DEBUG` on cluster

Reply:
Let's do this next time we increment a DIALECT.